<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
	border-image: url(:/other/resource/other/main.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_exit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(69,137,148);
border-radius:25px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_cal">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(69,137,148);
border-radius:25px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>计算器</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_dialog">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(69,137,148);
border-radius:25px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>对话自定义框</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_video">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(69,137,148);
border-radius:25px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>播放视频</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_weather">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(69,137,148);
border-radius:25px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>天气预报</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_say">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(167, 167, 255);</string>
    </property>
    <property name="text">
     <string>按住说话</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_clock">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(69,137,148);
border-radius:25px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>秒表</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
